core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -843,6 +845,45 @@ describe('Contributions and review component', function() {
       ctrl.switchToTab(ctrl.TAB_TYPE_CONTRIBUTIONS, 'add_question');
       ctrl.isActiveTab(ctrl.TAB_TYPE_CONTRIBUTIONS, 'add_question');
     });
+
+    it('should get option value using tab type ' +
+      'and suggestion type', function() {
+      spyOn(ctrl, 'activeDropdownTabChoice');
+      ctrl.activeDropdownTabChoice =
+        ctrl.getOptionValueFromTabAndSuggestionTypes(
+          ctrl.TAB_TYPE_CONTRIBUTIONS, 'add_question');
+
+      expect(ctrl.activeDropdownTabChoice).toBe(
+        ctrl.TAB_TYPE_CONTRIBUTIONS + ' ' + 'add_question');
+    });
+
+    it('should obtain tab type and suggestion type ' +
+      'from option value', function() {
+      ctrl.activeDropdownTabChoice =
+        ctrl.getOptionValueFromTabAndSuggestionTypes(
+          ctrl.TAB_TYPE_CONTRIBUTIONS, 'add_question');
+
+      var [tabType, suggestionType] =
```

Comment:
Prefer `const` over `var` if not reassigning the variable.

Reply:
I have removed this code while making changes with regard to another one of your comments, but I'll keep this in mind, thanks!

Reply:
Hmm I'm still the `var` here?

Reply:
Yes, that's because I haven't pushed any changes yet, I just wanted to ask what you thought of the dropdown.